app/controllers/application_controller.rb

```diff
@@ -198,8 +168,8 @@ def empty_params
     ActionController::Parameters.new
   end
 
-  def with_dropped_params(&block)
-    QuietStrongParams.with_dropped_params(&block)
+  def with_dropped_params(&)
+    QuietStrongParams.with_dropped_params(&)
```

Comment:
Was this change automatically done by the linter?

Reply:
Yes, I didn't change this line